samples/v1/updateFindingSourceProperties.js

```diff
@@ -54,6 +55,7 @@ function main(findingName = 'FULL_FINDING_PATH') {
   }
   updateFinding();
   // [END demo]
+  // [END securitycenter_demo]
```

Comment:
We seem to have collisions on this region tag, should it be different in each file?

Reply:
I do not know how I overlooked these. I'll handle this tomorrow - thank you and apologies!

Reply:
The NodeJS `demo` region tag exists in several repository files (with different sample code for each), and each sample is included across several documentation pages for Security Center. 

This PR was meant to start on the right path by getting all of them 'tracked' under Security Center product. The larger reality is that yes, really each one should be more explicit to the operations they perform (so we don't have the many conflicts we do). 

I hesitated to make that choice myself, but I am happy to if desired. I would likely wrap them based on the containing file, such as `securitycenter_updatefinding`. Afterwards, each documentation page will be updated to reflect the new(er) tag, then finally, I'll come through and clear out the unused tags.

Thoughts?

Reply:
Hey @Strykrol I think naming them after the operation in the file is a good starting point, to prevent collisions 👍

Reply:
Done!